src/connection/database/actions.js

```diff
@@ -181,7 +181,7 @@ function resetPasswordSuccess(id) {
   const m = read(getEntity, "lock", id);
   if (hasScreen(m, "login")) {
     swap(updateEntity, "lock", id, m => (
-      setScreen(l.setSubmitting(m, false), "login")
+      setScreen(l.setSubmitting(m, false), "login", [""])
```

Comment:
Add a comment why there is a `[""]` as a parameter since we will quickly forget and maybe add a `FIXME` so we fit it eventually